### PR TITLE
PRIME-514 - Angular 9

### PR DESCRIPTION
**Pull request by mtpultz**

### CODE REVIEW

File: prime-angular-frontend/Dockerfile [hunk 1]
```diff
@@ -19,37 +19,36 @@ ENV KEYCLOAK_CLIENT_ID $KEYCLOAK_CLIENT_ID
...
-    ng build --prod && \ 
+RUN npm install @angular/cli -g && \
+    npm install && \
+    npm audit fix && \
+    ng build --prod && \
     echo "NPM packages installed..."
 
-# RUN npm audit fix --only=prod && \ 
-#     npm install @angular-devkit/build-angular@0.803.24 \ 
+# RUN npm audit fix && \
```
Comment (hogmann) [inline prime-angular-frontend/Dockerfile]: Not sure if this is needed commented out here as we are including it above.

File: prime-angular-frontend/openshift.dockerfile [hunk 2]
```diff
@@ -36,15 +36,14 @@ COPY nginx.template.conf /etc/nginx/nginx.template.conf
...
+RUN cat /usr/src/app/src/environments/environment.prod.ts && \
+    npm install @angular/cli  -g --silent && \
+    npm install && \
+    npm audit fix && \
+    ng build --prod && \
     echo "NPM packages installed..."
 
-# RUN npm audit fix --only=prod && \ 
-#     npm install @angular-devkit/build-angular@0.803.24 \ 
+# RUN npm audit fix --only=prod && \
```
Comment (hogmann) [inline prime-angular-frontend/openshift.dockerfile]: Same here

File: prime-angular-frontend/Dockerfile (hunk 1 above)
Comment (mtpultz) [reply prime-angular-frontend/Dockerfile]: Yah, probably not, but it's Paul's code and I don't know if he had something in mind when he commented it out.  @poggdogg do we need this commented out code in the Dockerfile?

File: prime-angular-frontend/openshift.dockerfile (hunk 2 above)
Comment (mtpultz) [reply prime-angular-frontend/openshift.dockerfile]: @poggdogg do we need this commented out code in the Dockerfile?

**Comment by mtpultz** [comment]
@poggdogg I've updated the Dockerfile and OpenShift template.  I've removed our temporary patch and turned on audit fix in the other environments since I'd rather find out there's an issue with it updating something before we push to prod, or even test.

File: prime-angular-frontend/Dockerfile (hunk 1 above)
Comment (poggdogg) [reply prime-angular-frontend/Dockerfile]: Was commented out during troubleshooting of angular 8 - please delete.

**Pull request merged**